src/Bicep.LangServer.UnitTests/Handlers/BicepBuildCommandHandlerTests.cs

```diff
@@ -121,12 +121,12 @@ public async Task Handle_WithValidPath_AndErrorsAndWarningsInInputFile_ReturnsBu
 /input.bicep(3,2) : Error BCP007: This declaration type is not recognized. Specify a metadata, parameter, variable, resource, or output declaration.
 /input.bicep(3,2) : Error BCP001: The following token is not recognized: ""#"".
 /input.bicep(4,1) : Error BCP112: The ""targetScope"" cannot be declared multiple times in one file.
-/input.bicep(4,13) : Error BCP018: Expected the ""="" character at this location.
-/input.bicep(4,13) : Error BCP009: Expected a literal value, an array, an object, a parenthesized expression, or a function call at this location.
+/input.bicep(4,12) : Error BCP018: Expected the ""="" character at this location.
+/input.bicep(4,12) : Error BCP009: Expected a literal value, an array, an object, a parenthesized expression, or a function call at this location.
 /input.bicep(6,2) : Error BCP007: This declaration type is not recognized. Specify a metadata, parameter, variable, resource, or output declaration.
 /input.bicep(6,2) : Error BCP001: The following token is not recognized: ""#"".
 /input.bicep(7,1) : Error BCP112: The ""targetScope"" cannot be declared multiple times in one file.
-/input.bicep(7,15) : Error BCP009: Expected a literal value, an array, an object, a parenthesized expression, or a function call at this location.
+/input.bicep(7,14) : Error BCP009: Expected a literal value, an array, an object, a parenthesized expression, or a function call at this location.
```

Comment:
Thanks for including these. I keep having to turn "trim trailing whitespace" is VS Code off when a PR touches this file.